android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -1,40 +1,58 @@
 package io.sentry.capacitor;
 
+import com.getcapacitor.JSArray;
+import com.getcapacitor.JSObject;
 import com.getcapacitor.NativePlugin;
 import com.getcapacitor.Plugin;
 import com.getcapacitor.PluginCall;
 import com.getcapacitor.PluginMethod;
 
+import io.sentry.Breadcrumb;
 import io.sentry.Integration;
+import io.sentry.Sentry;
+import io.sentry.SentryLevel;
 import io.sentry.SentryOptions;
 import io.sentry.UncaughtExceptionHandlerIntegration;
 import io.sentry.android.core.AnrIntegration;
 import io.sentry.android.core.NdkIntegration;
 import io.sentry.android.core.SentryAndroid;
 import io.sentry.protocol.SdkVersion;
-import io.sentry.protocol.SentryException;
 
 import java.io.File;
 import java.io.FileOutputStream;
+import java.io.UnsupportedEncodingException;
 import java.nio.charset.Charset;
 import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.UUID;
 
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.util.Log;
+
 @NativePlugin
 public class SentryCapacitor extends Plugin {
 
-    static final Logger logger = Logger.getLogger("capacitor-sentry");
+    final static Logger logger = Logger.getLogger("capacitor-sentry");
     private SentryOptions sentryOptions;
+    private Context context;
+    private static PackageInfo packageInfo;
 
-    // @PluginMethod
-    // public void echo(PluginCall call) {
-    //     String value = call.getString("value");
+    @Override
+    public void load() {
+        super.load();
 
-    //     JSObject ret = new JSObject();
-    //     ret.put("value", value);
-    //     call.success(ret);
-    // }
+        if (this.context == null) {
+            this.context = this.bridge.getContext();
+            try {
+                this.packageInfo = this.context.getPackageManager().getPackageInfo(this.getContext().getPackageName(), 0);
```

Comment:
Does this `packInfo` not need to get set if `context` already exists?

Reply:
ahh, good catch. I'll move it outside that if statement so packageInfo is set at all times.